assertj-core/src/main/java/org/assertj/core/api/AbstractOffsetDateTimeAssert.java

```diff
@@ -854,8 +854,8 @@ public SELF isInTheFuture() {
    *                                          .isBetween(parse("2010-01-01T01:00:00+01:00"),
    *                                                     parse("2010-01-01T01:00:00+01:00"));</code></pre>
    *
-   * @param startExclusive the start value (exclusive), expected not to be null.
-   * @param endExclusive the end value (exclusive), expected not to be null.
+   * @param startInclusive the start value (exclusive), expected not to be null.
+   * @param endInclusive the end value (exclusive), expected not to be null.
```

Comment:
change `(exclusive)` -> `(inclusive)` too ;-)

Reply:
ah, overlooked: 32ca67a6c152b85f7cff4491229bb0854b487d6e

Reply:
sorry, i didn't get, i thought it's already updated: https://github.com/assertj/assertj/pull/3217/files#diff-392694e0c64f8c2de5b901842eb91aef6fad7a410c34fb7faeedf7ba9e5e6686R895-R908

Or you mean smth else?